### ISSUE TITLE
DeltaChat Desktop messages are not readable on the Android II app

### ISSUE THREAD

**Issue by sbadux** [deltachat/deltachat-android#459]
Hello,I've tried to chat between DeltaChat Desktop (Ubuntu 18.04) and DeltaChat Android II (Android 8.0, Deltachat v.0.0.2). The desktop version can read/send messages, but the Android II version can only send messages. The replies from the Desktop are not readable and them are shown as .bin attachments

This is what I see from the Desktop version
![image](https://user-images.githubusercontent.com/28478027/46909469-486a2e00-cf33-11e8-99f0-9bfcc2789cae.png)

This is what I see from the Android II version
![screenshot_20181013-215339](https://user-images.githubusercontent.com/28478027/46909466-34bec780-cf33-11e8-8cbe-be39d759a461.png)

**Comment by angelo-fuchs** [comment]
@r10s This issue looks stale (and not up to date anymore) I'll close it. Feel free to reopen as needed.

**Comment by wildcart** [comment]
@r10s  I am sorry to reply to an old / closed issue but I wasn't sure if I should have opened a new issue:

I am seeing the exact same behaviour between an Android 7.0 device sending messages and an Android 8.1 device and the desktop app on the receiving side. Both Android apps use the same version: 0.304.0, the desktop app is version 0.102.0. The Android 7 device is configured to use a Hotmail E-Mail account, while the Android 8.1/desktop app use a Strato Account
* Android 7 (encrypted) to Android 8.1 / desktop --> fails; shows two messages instead; ATT00001 ATT00002.bin 
* Android 7 (unencrypted) to Android 8.1 / desktop --> works
* Android 8.1 / desktop (encrypted) to Android 7 --> works
* Android 8.1 / desktop (unenctyped) to Android 7 --> works

The problem is the same as described above. If the Android 7 app sends encrypted messages the Android 8.1 / desktop receives two messages which are displayed as shown in the screenshots above.

I have some encrypted chats with other contacts that work fine and looking at the messages I receive on the Mail server I see a clear difference between the messages:
* Encrypted messages received from another contact that can be read by the Android 8.1 app have the following two attachments: Part_1.pgp and Part_2.bin
* Encrypted messages received from the Android 7 app that cannot be read by the Android 8.1 app have the following two attachments: ATT00001 ATT00002.bin

I downloaded the attachment and looking at the content I don't see much of a difference. Both seem to contain validly PGP encrypted texts. I'll try to export my private key from the Android 8.1 device to check if I can decrypt the text in ATT00002.bin manually.

Unfortunately, I do not have access to the Android 7 device nor to the Hotmail account.

One more things (may be unrelated): The chat was initiated from the Android 7 device, but the Android 8.1 device never received the first message nor a 'contact request'. Only after sending a message from the Android 8.1 device the chat was established (the initial Android 7 message was never received, it's also not available on the Strato Mail account).


Is there anything else I can do to help to figure out what's going on?

best
  Christian.

**Comment by r10s** [comment]
hi @wildcart I think this is the same issue as in https://github.com/deltachat/deltachat-core/issues/561 - in general, we cannot recommend using hotmail or outlook addresses currently (exchange, however, seems to work)

**Comment by wildcart** [comment]
@r10s thanks for linking the issue. No idea why I didn't see the other while browsing through the list of open issues :(

**Comment by r10s** [comment]
> No idea why I didn't see the other while browsing through the list of open issues :(

there are jus too many issues :)
